.github/workflows/ci-turbo.yml

```diff
@@ -98,3 +97,15 @@ jobs:
               env:
                   NODE_OPTIONS: --max-old-space-size=8192
                   TURBO_SCM_BASE: ${{ env.TURBO_SCM_BASE }}
+
+    posthog-analytics:
+        name: PostHog CI analytics
+        if: always()
+        needs: [turbo-affected-4] # last job in this workflow
```

Comment:
Keeping the last here is tricky to keep up to date, right? What if we listed *all* jobs somehow? Could even have a CI action (inception) that checks whether this is correct

Reply:
makes a lot of sense :)
I'm sure the new CI analytics product will be a hit once we have that 😜